torch/testing/_asserts.py

```diff
@@ -751,8 +808,7 @@ def assert_close(
 
     Raises:
         UsageError: If a :class:`torch.Tensor` can't be constructed from an array-or-scalar-like.
-        UsageError: If any tensor is quantized or sparse. This is a temporary restriction and will be relaxed in the
-            future.
+        UsageError: If any tensor is sparse. This is a temporary restriction and will be relaxed in the future.
```

Comment:
Does this need an additional paragraph to describe quantized tensor comparisons?